src/lib/utils/utils.js

```diff
@@ -620,6 +620,27 @@ function toposort(graph) {
   return toposortGraph(graph);
 }
 
+function getInitProviderCode(plugins, configs) {
```

Comment:
Do you think it could be the responsability of the code_generator to return that?
(via event)

Reply:
Probably better, yes, will update.